firefox-ios/Client/Frontend/Settings/ContentBlockerSettingViewController.swift

```diff
@@ -185,4 +194,11 @@ class ContentBlockerSettingViewController: SettingsTableViewController {
     func done() {
         settingsDelegate?.didFinish()
     }
+
+    // MARK: - ThemeApplicable
+    func applyTheme(theme: Theme) {
+        let colors = theme.colors
+        linkButton.setTitleColor(colors.actionPrimary, for: .normal)
```

Comment:
we should be able to remove this now, and since we have an accessing `applyTheme`, what do you think about using the `override` method?